src/applications/virtual-agent/components/webchat/startConvoAndTrackUtterances.js

```diff
@@ -0,0 +1,47 @@
+import {
+  processActionConnectFulfilled,
+  processSendMessageActivity,
+  processIncomingActivity,
+} from './helpers/startConvoAndTrackUtterancesHelpers';
+
+const StartConvoAndTrackUtterances = {
+  makeBotStartConvoAndTrackUtterances: (
+    csrfToken,
+    apiSession,
+    apiURL,
+    baseURL,
+    userFirstName,
+    userUuid,
+    requireAuth,
```

Comment:
`requireAuth` is a deprecated feature toggle from the earlier part of 2021. We are no longer using it for its intended purpose, and it has been on since March 2021, so we can remove any reference to it

Reply:
removed